plans/tier0.fmf

```diff
@@ -4,6 +4,11 @@ description+: |
     Critical set of tests/plans to be run for each upstream PR
     before merge commit to the main branch.
 
+adjust+:
+  - enabled: false
+    when: distro == stream-8-latest
```

Comment:
I don't think this is necessary, but it won't hurt to keep it this way.

https://fmf.readthedocs.io/en/latest/context.html#equality-vs-comparison